### ISSUE TITLE
Add example for new Babel 7 TypeScript Integration

### ISSUE THREAD

**Issue by fabb** [TypeStrong/fork-ts-checker-webpack-plugin#171]
Babel 7 supports TypeScript without type checking, so basically it just strips all the types away and makes the code valid JS code. It still would be preferrable to do type checking with `ts` in a separate process during the build. It would ve nice if in the readme of this project there was an example on how to configure this correctly (if it is possible).

**Comment by antch** [comment]
I think the same pattern could be applied for linting along with `AngularCompilerPlugin` as well. I use `ForkTsCheckerWebpackPlugin` for non-Angular TypeScript files but I'm stuck.

**Comment by kenjiru** [comment]
Does this plugin even work with Babel 7? `webpack --watch` doesn't work anymore when enabling this plugin.

**Comment by johnnyreilly** [comment]
I believe that create-react-app are using it - so could be worth ejecting one of their apps to get config.

**Comment by huy-nguyen** [comment]
I do use babel 7's typesceript transpilation and this is my settings for development. For production, I run `tsc --noEmit` instead of using the plugin.
```javascript
import ForkTSCheckerWebpackPlugin from 'fork-ts-checker-webpack-plugin';
// ...
export default () => {
  return {
    module: {
      rules: [
        {
          test: /\.(ts|tsx)$/,
          use: [
            loader: 'babel-loader',
            options: {
              presets: [
                '@babel/preset-typescript',
              ]
            }
          ],
          exclude: /node_modules/,
        } 
      ],
      // ... other rules
    },
    plugins: [
      new ForkTSCheckerWebpackPlugin({
        checkSyntacticErrors: true,    
      }),
      // ... other plugins
    ]
  }
}
```

**Comment by bestseob93** [comment]
oops.. this was typescript problem.
typescript version: 3.4.1 is very slow.
